selvpc/servers.go

```diff
@@ -19,3 +25,11 @@ func serversMapsFromStructs(serverStructs []servers.Server) []map[string]interfa
 
 	return associatedServers
 }
+
+// hashServers is a hash function to use with the "servers" set.
+func hashServers(v interface{}) int {
+	var buf bytes.Buffer
+	m := v.(map[string]interface{})
+	buf.WriteString(fmt.Sprintf("%s-", m["id"].(string)))
```

Comment:
Why is bytes.Buffer used instead of just using fmt.Sprintf to generate a string?

Reply:
Well, I just wanted to use same way in all `hashSomething` functions. But I agree, it's better to use `fmt.Sprintf` in cases where we don't check if attribute is set.
I will fix that in `hashServers` and `hashQuotas` functions.